examples/discovery-search-app/scripts/runExampleApp.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -e
+
+PREREQUISITES=("yarn" "jq")
+SCRIPT_DIR=$(
+    cd "$(dirname "$0")"
+    pwd
+)
+USE_FORMAT=$(
+  type tput >/dev/null 2>&1
+  if [ $? -eq 0 ]; then echo "true"; else echo "false"; fi
+)
+
+function boldMessage() {
+  if [ "$USE_FORMAT" == "true" ]; then
+    echo "$(tput bold)$1$(tput sgr 0)"
+  else
+    echo "$1"
+  fi
+}
+
+function colorMessage() {
+  if [ "$USE_FORMAT" == "true" ]; then
+    echo "$(tput setaf $2)$1$(tput sgr 0)"
+  else
+    echo "$1"
+  fi
+}
+
+function paddedMessage() {
+  echo
+  echo
+  boldMessage "$1"
+}
+
+function checkForDependencies() {
+  boldMessage "Checking for prerequisites..."
+  missing=0
+  for i in "${PREREQUISITES[@]}"; do
+    echo -n "  $i: "
+
+    set +e
+    type $i >/dev/null 2>&1
+    exists=$?
+    set -e
+
+    if [ $exists -eq 0 ]; then
+      colorMessage "Y" 2
+    else 
+      colorMessage "N" 1
+    fi
+
+    (( missing += exists ))
+  done
+
+  if [[ $missing > 0 ]]; then
+    echo
+    echo "Not all prerequistes are installed." >&2
+    exit 1
+  fi
+}
+
+function updateEnvFile() {
+  paddedMessage "Provide the following based on the Discovery workspace url template:"
+  colorMessage "  https://{cluster_host}:{cluster_port}/discovery/{release_name}/projects/{project_id}/workspace" 3
+  echo
+
+  read -p "  cluster_host: " host
+  read -p "  cluster_port (e.g. 443): " port
+  read -p "  project_id: " projectId
+
+  paddedMessage "CP4D credentials:"
+  echo "  These are the credentials used to log into the CP4D dashboard"
+  echo
+  read -p "  username: " username
+  read -sp "  password: " password
+
+  if [[ -z $host || -z $port || -z $username || -z $password ]]; then
+    echo
+    echo
+    echo "You provide all of the following: host, port, username and password" >&2
+    exit 1
+  fi
+
+  cat >"$SCRIPT_DIR/../.env.local" <<EOL
```

Comment:
When users want to set up Discovery Components in their own applications, are they expected to populate `.env` or `.env.local`? If we want them to put their credentials in `.env` at some point, it might be a good idea to call out in the readme that this script only populates `.env.local`.

Reply:
If we switch `.env` -> `.env.template`, that would ease confusion around this point